src/app/mytools/mytools.service.ts

```diff
@@ -95,21 +115,49 @@ export class MytoolsService extends MyEntriesService<DockstoreTool, OrgToolObjec
     return entry.tool_path || '';
   }
 
-  sortEntry(entryA: DockstoreTool, entryB: DockstoreTool): number {
-    const registryA = entryA.registry_string.toLowerCase();
-    const registryB = entryB.registry_string.toLowerCase();
-    const compareRegistry = registryA.localeCompare(registryB);
+  sortEntry(entryA: DockstoreTool | Workflow, entryB: DockstoreTool | Workflow): number {
```

Comment:
wow

Reply:
This method is pretty complex. Suggestions:

1. Add test(s) in the spec file
2. Use methods/user guards for your if statements (`if (isAppTool(entryA))` is easier to understand than `if ('full_workflow_path' in entryA)`)